help/destinations/catalog/email-marketing/mailchimp-tags.md

```diff
@@ -0,0 +1,173 @@
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Mailchimp Tags] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. 
+
+To correctly map your XDM fields to the [!DNL Mailchimp Tags] destination fields, follow the steps below:
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select identity namespace]** and select `Email`.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select `Email`.
+    The mappings between your XDM profile schema and [!DNL Mailchimp Tags] will be as below:
+    | Source Field | Target Field | Mandatory |
+    | --- | --- | --- |
+    |`IdentityMap: Email`|`Identity: email`| Yes |
+
+    An example with the above mappings is shown below:
+    ![Platform UI screenshot example showing Target mappings.](../../assets/catalog/email-marketing/mailchimp-tags/mappings.png)
+
+When you have finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Log in to the [[!DNL Mailchimp]](https://login.mailchimp.com/) website. Then navigate to the **[!DNL Audience]** > **[!DNL All Contacts]** page and check if the contacts from the segment have been added and contacts within the segment have been updated with the segment name.
+![Mailchimp UI screenshot showing the Audience page.](../../assets/catalog/email-marketing/mailchimp-tags/contacts.png)
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](/help/data-governance/home.md).
+
+## Errors and troubleshooting {#errors-and-troubleshooting}
+
+Refer to the [[!DNL Mailchimp] Errors](https://mailchimp.com/developer/marketing/docs/errors/) page for a comprehensive list of status and error codes with explanations.
```

Comment:
Refer to the [!DNL Mailchimp] [errors page](https://mailchimp.com/developer/marketing/docs/errors/)